tests/service/predictors/srl_test.py

```diff
@@ -46,4 +46,11 @@ def test_cpu_vs_gpu(self):
             prediction_cpu = predictor_cpu.predict_json({"sentence": sentence})
             prediction_gpu = predictor_gpu.predict_json({"sentence": sentence})
 
-            assert prediction_cpu == pytest.approx(prediction_gpu)
+            assert set(prediction_cpu.keys()) == {"verbs"}
```

Comment:
In an ideal world we would be able to test that loading models cross device provides the same result, but that's going to be difficult given we don't have GPU backed tests. Either create this test and skip it for now, or yeah, just remove it.